pkg/sloop/server/internal/testFiles/testConfig.txt

```diff
@@ -0,0 +1,44 @@
+{
+	"ConfigFile": "",
+	"leftBarLinks": null,
+	"resourceLinks": null,
+	"disableKubeWatch": false,
+	"kubeWatchResyncInterval": 1800000000000,
```

Comment:
consider using something like `180m` instead of some large number of nanoseconds
repeat for other `time.Duration` values